### PR TITLE
cleaned up overall design and linked about view to main content view

### PR DESCRIPTION
**Pull request by SPJ425cascadia**

### CODE REVIEW

**Review by adamkohler — APPROVED** [review]
Sean my only concern is the blue text buttons on the bottom of the About view, down the road we may want to change the color scheme so the text stands out better from the background but im good for this to be merged.

**Review by antondtran — APPROVED** [review]

File: YUP253/About.swift [hunk 1]
```diff
@@ -33,6 +33,7 @@ struct About: View {
```
Comment (gomezfox) [inline YUP253/About.swift:33]: Padding of 200 overflows screen on older devices, 
* Recommend changing the Image padding to a smaller value (the desired top padding)
* Then to automatically line the rest of the items at the bottom, use the `Spacer(minLength: 0)` in line 32

**Review by gomezfox — CHANGES_REQUESTED** [review]
Overflows vertically on older devices, may be solved using `Spacer`

**Review by jdlacy — APPROVED** [review]
Looks fine

**Review by gomezfox — APPROVED** [review]

**Pull request merged**